.github/workflows/nominatim-docker-test.yml

```
@@ -0,0 +1,36 @@
+name: nominatim-docker-test
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  
+  push:
+    paths:
+      - 'net/ext_service/geocoder'
+      - 'analysis/intake/cleaning/clean_and_resample.py'
+
```

we want to test on pull requests for these paths as well

```suggestion
  push:
    paths:
      - 'net/ext_service/geocoder'
      - 'analysis/intake/cleaning/clean_and_resample.py'

  pull_request:
    paths:
      - 'net/ext_service/geocoder'
      - 'analysis/intake/cleaning/clean_and_resample.py'
```